agents/a2a_mcp_integration.py

```
@@ -427,6 +425,71 @@ async def handle_collaboration_request(self, message: A2AMessage) -> Dict[str, A
+                        "params": {"name": tool_name, "arguments": params},
+                        "id": 1,
+                    },
+                    timeout=aiohttp.ClientTimeout(total=30),
+                ) as response:
+                    if response.status == 200:
+                        result = await response.json()
+                        return result.get("result", {})
+                    else:
+                        logger.error(
+                            f"MCP tool call failed: {
+                                response.status}"
```

Ruff (0.11.9)

Logging statement uses f-string

(G004)